lib/puppet/provider/package/brewcask.rb

```diff
@@ -27,7 +27,7 @@ def self.caskroom
   end
 
   def self.current(name)
-    caskdir = Pathname.new "#{caskroom}/#{name}"
```

Comment:
This needs to stay as double quotes otherwise it won't be evaluated and interpolated correctly.